web/CMakeLists.txt

```diff
@@ -1,78 +1,45 @@
 project(trweb)
 
-set(TRWEB_SRCS
-  src/about-dialog.js
-  src/action-manager.js
-  src/alert-dialog.js
-  src/context-menu.js
-  src/file-row.js
-  src/formatter.js
-  src/inspector.js
-  src/main.js
-  src/move-dialog.js
-  src/notifications.js
-  src/open-dialog.js
-  src/overflow-menu.js
-  src/prefs-dialog.js
-  src/prefs.js
-  src/remote.js
-  src/remove-dialog.js
-  src/rename-dialog.js
-  src/shortcuts-dialog.js
-  src/statistics-dialog.js
-  src/torrent.js
-  src/torrent-row.js
-  src/transmission.js
-  src/utils.js
-  style/transmission-app.scss
+file(GLOB_RECURSE TRWEB_ASSETS
```

Comment:
Using `file(GLOB)` to populate source files list breaks tracking of new files added after the initial configuration. CMake 3.12 added a `CONFIGURE_DEPENDS` option to this command to help a little (and I'm open to using it, even though it's still not recommended), but our top-level CMakeLists.txt currently sets the minimum at 3.9 (see my comment at https://github.com/transmission/transmission/pull/2063#pullrequestreview-793692566 about that).

Reply:
DYKI the CI boxen all have >= 3.12?

Easy to one-time populate the list with the existing files. Slightly harder to remember to do that in the future if/when we add or remove a file :smiley_cat: